music_assistant/server/controllers/music.py

```diff
@@ -360,6 +360,14 @@ async def add_item_to_favorites(
             full_item.item_id,
             True,
         )
+        # ensure item is added to provider library
+        for provider_mapping in full_item.provider_mappings:
+            prov_controller = self.mass.get_provider(provider_mapping.provider_instance)
+            if prov_controller and prov_controller.library_edit_supported(full_item.media_type):
+                with suppress(NotImplementedError):
```

Comment:
```suggestion
```
Just remove this and fix the indentation and its all fixed already.
With the comment that the item will be added to the library of ALL INSTANCES of a provider and I can imagine people not wanting that